app/controllers/issues_controller.rb

```diff
@@ -26,6 +26,8 @@ def new
     end
 
     @line_type = params[:line_type]
+    @line_id = params[:line_id]
+    @stop_id = params[:stop_id]
```

Comment:
You added this instance variable, but I don't see it used anywhere. Am I missing something?